### DISCUSSION TITLE
How can I earn galaxy brain badge[mark]🥺

### DISCUSSION THREAD

**Discussion by Akash-Gupta-git** [recodehive/Stackoverflow-Analysis#430] [Q&A] [answered]
I have done my part.

Helping people, answering their questions, asking questions etc.

But I didn't get this badge

**Comment by sanjay-kv** [comment, marked as answer]
https://stream.recodehive.com/github

**Comment by Akash-Gupta-git** [comment]
I also saw that video in GSSOC academy

There is an option to give marks if someone gives marks

Can someone give marks to my answers
